client/apps/city_issues/views.py

```diff
@@ -187,3 +187,27 @@ def get_context_data(self, **kwargs):
         context = super(CheckIssues, self).get_context_data(**kwargs)
         context['issues_range'] = range(context["paginator"].num_pages)
         return context
+
+
+class CommentIssues(CreateView):
+    """Comment issue"""
+    template_name = 'comment_issue.html'
+    model = Comments
+    fields = ['comment']
+
+    def get_context_data(self, **kwargs):
+        context = super(CommentIssues, self).get_context_data(**kwargs)
+        context['issue'] = Issues.objects.get(pk=self.kwargs['pk'])
+        return context
+
+    def form_valid(self, form):
+        form = form.save(commit=False)
+        issue = Issues.objects.get(pk=self.kwargs['pk'])
+        user = User.objects.get(pk=self.request.user.id)
```

Comment:
You dont check if the user is authenticated. Server returns error `User matching query does not exist.`

Reply:
fixed

Reply:
fixed add mixin check auth

Reply:
look into mixins.py

Reply:
mixins approach, cool

Reply:
I know that I cool)